eos-chrome-plugin-update-upstream

```diff
@@ -45,6 +45,9 @@ WIDEVINE_FILENAME=
 
 WIDEVINE_VERSION=
 WIDEVINE_SO_FILE=libwidevinecdm.so
+WIDEVINE_TARGET_SO_FILE=libwidevinecdm.so
```

Comment:
I don't think this new variable is actually necessary since WIDEVINE_SO_FILE is assigned once above and never again, though I see that it is for consistency with the manifest.

Reply:
True true, I just did it for consistency indeed.